migrations/AttendanceIncentivizer.sol

```diff
@@ -0,0 +1,87 @@
+pragma solidity ^0.4.11;
+
+contract AttendanceIncentivizer {
+  event Donate(address indexed donater, uint indexed amount);
+  event Payout(address indexed donater, uint indexed amount);
+  event Validate(address indexed validator, address indexed validatee);
+
+  mapping (address => uint) public donaters;
+  mapping (address => uint) public validations;
+  address[] public donaterList;
+  address public owner;
+  address public winner;
+  uint constant minimumDonation = 1;
+  uint totalContractValue = 0;
+  int256 i=0;
+
+  function AttendanceIncentivizer() public payable {
+    owner = msg.sender;
```

Comment:
I see we have an owner field, but we're not restricting anything to `onlyowner`

Reply:
Agree. Don't think there's currently any need for owner. Will remove.